res/controllers/Traktor-Kontrol-S4-MK3.js

```diff
@@ -40,95 +40,102 @@ const KeyboardColors = [
 
 /*
  * USER CONFIGURABLE SETTINGS
- * Adjust these to your liking
+ * Change settings in the preferences (Mixxx >= 2.5)
```

Comment:
no need to specify version since by definition the user will be on a valid version